youtube_dl/extractor/xhamster.py

```diff
@@ -25,8 +25,8 @@ class XHamsterIE(InfoExtractor):
                     https?://
                         (?:.+?\.)?%s/
                         (?:
-                            movies/(?P<id>\d+)/(?P<display_id>[^/]*)\.html|
-                            videos/(?P<display_id_2>[^/]*)-(?P<id_2>\d+)
+                            movies/(?P<id>[\dA-z]+)/(?P<display_id>[^/]*)\.html|
+                            videos/(?P<display_id_2>[^/]*)-(?P<id_2>[\dA-z]+)
```

Comment:
I've already pointed out: if you include any non alphanumeric characters add tests for such cases.

Reply:
My bad, i meant for it to only be a alphanumeric regex

Reply:
Do I see it correctly that the problem is that a test for the movie URLs is still missing? 

Unfortunately I have not seen any URLs with included ../movie/... for a long time. The video ID with letters is only since 1 to 2 months.

Can't we maybe first make the change for ../videos/... until someone finds a suitable link for the test with ../movies/...?